log/loggerInterface.go

```diff
@@ -24,7 +24,7 @@ type Logger interface {
 }
 
 // LogEntry describes an entry to log.
-type LogEntry struct {
+type LogEntry struct { //nolint:golint
```

Comment:
This eludes me as to why this is needed.

My personal rule for lint exclusions is that they should be accompanied by a comment justifying them.

Reply:
I had a comment.
It is because of stuttering.
log.LogEntry.
I tried: `// log.LogEntry stutters //noling:golint`
and `//noling:golint log.LogEntry stutters`
both didn't work.
I didn't want to put it with the type doc cause that would be rendered as godoc.

Any other suggestions?

Reply:
nvm `//nolint:golint //reason` works.... and there is even a linter for this: 
https://github.com/golangci/golangci-lint/blob/master/vendor/github.com/go-critic/go-critic/checkers/whyNoLint_checker.go
:)

Reply:
Done

Reply:
That's cool. Thanks. It's a shame that the linters are not named very well. "golint" doesn't say much. If it was `//nolint:stutter` then it would not need an explanation I think.